src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/DwpUploadResponseHandler.java

```diff
@@ -47,13 +48,44 @@ public void handle(CallbackType callbackType, Callback<SscsCaseData> callback) {
             throw new IllegalStateException("Cannot handle callback");
         }
 
-        boolean dwpFurtherInfo =
+        BenefitType benefitType = callback.getCaseDetails().getCaseData().getAppeal().getBenefitType();
```

Comment:
Probably not relevant,  but worth asking the question.   Will we always have an appeal, and benefit type and a benefit type code ?    O/W will get NPEs here.   Maybe worth adding null checks just to be sure